templates/zerver/emails/organisation_info.source.html

```diff
@@ -0,0 +1,21 @@
+{% extends "zerver/emails/compiled/email_base_default.html" %}
+
+{% block illustration %}
+<img src="{{ email_images_base_uri }}/email_logo.png" alt=""/>
+{% endblock %}
+
+{% block content %}
+<p>{{ _('New organization anu malik songsdetails') }}</p>
+
+<p>
+    {% if realm_creation %}
+        {% trans %} Organisation Name : <b>{{ realm_name }}</b>.{% endtrans %}
```

Comment:
Organization -- we use American spelling.

Reply:
Shouldn't this be marked for translation like the other strings such as ``_('Organization URL:')``?